nsxt/resource_nsxt_policy_distributed_vlan_connection.go

```diff
@@ -34,6 +34,7 @@ var distributedVlanConnectionSchema = map[string]*metadata.ExtendedSchema{
 			SdkFieldName: "VlanId",
 		},
 	},
+	"subnet_exclusive_config": metadata.GetExtendedSchema(subnetExclusiveConfigSchema()),
```

Comment:
Hi Arun, is there support for this new attribute needs to be coded manually?
Looking at the schema, I would expect it to work if defined with `Metadata`, in which case both write and read conversions will be supported out of the box.